src/index.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App';
+import 'bootstrap/dist/css/bootstrap.css';
+import * as actions from './actions/book';
+import store from './reducers/index';
+// import configureStoreApp from './reducers/configureStore';
+
+// const store = configureStoreApp();
```

Comment:
- [ ] <b>OPTIONAL:</b> To make sure that you always keep your code clean, remember to remove all commented-out code when you don't need them any more.